app/views/foreman/job_templates/change_content_source.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Change content source
+job_category: Katello
+model: JobTemplate
+provider_type: SSH
+description_format: Configure subscription manager to new content source (smart proxy)
+feature: katello_change_content_source
+template_inputs:
+- name: smart_proxy
```

Comment:
The name can be human friendly, e.g. "Smart Proxy", then below in the template you refer to it as <%= input("Smart Proxy") %>. The name is used in the form and users should not see UI with underscores.

Also I'd consider renaming it to Capsule (as mentioned below)